deploy.yaml

```diff
@@ -0,0 +1,36 @@
+#Using this will require buildcomponents.yaml  
+
+
+before_script:
```

Comment:
I don't think there's actually a good place for this before_script, but I don't think its deploy, if you are creating a package CI file, you're not going to need deploy, but you would need this before_script.